Configuration/Realm/Tests.xcconfig

```diff
@@ -1,10 +1,11 @@
-SUPPORTED_PLATFORMS = macosx iphonesimulator iphoneos;
+SUPPORTED_PLATFORMS = macosx iphonesimulator iphoneos appletvos appletvsimulator;
 SKIP_INSTALL = YES;
 
 INFOPLIST_FILE = Realm/Tests/RealmTests-Info.plist;
 PRODUCT_BUNDLE_IDENTIFIER = io.Realm.${PRODUCT_NAME:rfc1034identifier};
 
 LD_RUNPATH_SEARCH_PATHS[sdk=iphone*] = $(inherited) @executable_path/Frameworks @loader_path/Frameworks;
+LD_RUNPATH_SEARCH_PATHS[sdk=appletv*] = $(inherited) @executable_path/Frameworks @loader_path/Frameworks;
 LD_RUNPATH_SEARCH_PATHS[sdk=macosx*] = $(inherited) @executable_path/../Frameworks @loader_path/../Frameworks;
```

Comment:
Part of the reason I ask is that the same setting was also absent from Realm.xcconfig. Perhaps we've just not had reports of it causing an issue yet since I'm guessing it may be a regression on master from my Xcode project refactoring.